test/e2e/framework/volume_util.go

```diff
@@ -356,6 +356,34 @@ func StartVolumeServer(client clientset.Interface, config VolumeTestConfig) *v1.
 	return pod
 }
 
+// Wrapper of cleanup function for volume server without secret created by specific CreateStorageServer function.
+func CleanUpVolumeServer(f *Framework, serverPod *v1.Pod) {
+	CleanUpVolumeServerWithSecret(f, serverPod, nil)
+}
+
+// Wrapper of cleanup function for volume server with secret created by specific CreateStorageServer function.
+func CleanUpVolumeServerWithSecret(f *Framework, serverPod *v1.Pod, secret *v1.Secret) {
+	cs := f.ClientSet
+	ns := f.Namespace
+
+	var secErr error
+	if secret != nil {
+		Logf("Deleting server secret %q...", secret.Name)
+		secErr = cs.CoreV1().Secrets(ns.Name).Delete(secret.Name, &metav1.DeleteOptions{})
+	}
+
+	Logf("Deleting server pod %q...", serverPod.Name)
+	err := DeletePodWithWait(f, cs, serverPod)
+	if secErr != nil || err != nil {
+		if secErr != nil {
```

Comment:
Please move to `cs.CoreV1().Secrets(ns.Name).Delete()` and leave only `if err != nil` here. I'd also suggest using `err` instead of `secErr` everywhere.